### ISSUE TITLE
Attaching a file causes UnrecognizedExceptionException

### ISSUE THREAD

**Issue by alphametric** [php-webdriver/php-webdriver#746]
### What are you trying to achieve? (Expected behavior)
Attach a file using a standard `<input type="file">`

### What do you get instead? (Actual behavior)
I receive an `UnrecognizedExceptionException`

```bash
Facebook\WebDriver\Exception\UnrecognizedExceptionException: invalid argument: 'text' is empty
  (Session info: chrome=79.0.3945.130)
  (Driver info: chromedriver=79.0.3945.36 (3582db32b33893869b8c1339e8f4d9ed1816f143-refs/branch-heads/3945@{#614}),platform=Mac OS X 10.15.3 x86_64)

/vendor/php-webdriver/webdriver/lib/Exception/WebDriverException.php:158
/vendor/php-webdriver/webdriver/lib/Remote/HttpCommandExecutor.php:331
/vendor/php-webdriver/webdriver/lib/Remote/RemoteWebDriver.php:565
/vendor/php-webdriver/webdriver/lib/Remote/RemoteExecuteMethod.php:40
/vendor/php-webdriver/webdriver/lib/Remote/RemoteWebElement.php:339
```

**However** using 1.8@dev addresses the issue.

### Details
* Php-webdriver version: 1.7.1
* PHP version: 7.4.2
* Operating system: Mac OS Catalina
* Browser used + version: Chrome 79

**Comment by OndraM** [comment]
> However using 1.8@dev addresses the issue.

So then it is already fixed I guess, and 1.8.0 will be released soon :).

**Comment by alphametric** [comment]
Ah, okay great. Thanks! Wasn't sure how long it would be before 1.8 was out, so opened the issue in case 1.7 needed a patch.
